docs/wallet/wallet-token.md

```diff
@@ -0,0 +1,74 @@
+---
+id: create-a-fungible-token
+title: Web Wallet Tokens
+---
+
+## **Introduction**
+
+**ESDT** stands for *Elrond Standard Digital Token*.
+
+Elrond network natively supports the issuance of custom tokens, without the need for contracts such as ERC20, but addressing the same use-cases.
+You can create and issue an ESDT token from [Elrond web wallet](https://wallet.elrond.com/) in a few steps. Let's go over these steps.
+
+## **Prerequisites**
+
+- A wallet on Elrond Network.
+- 0.05 EGLD issuance fee
+- fees for transactions
+
+## **Creating a fungible token from Web Wallet**
+
+To get started, open up the [Elrond web wallet](https://wallet.elrond.com/). You can create a new wallet if you do not have one or import your existing wallet. Here is a [guide](https://docs.elrond.com/wallet/web-wallet/) to help you navigate.
+
+On the left sidebar, from the **ISSUE** section click on **Tokens**.
```

Comment:
could add a screenshot with the menu item highlighted in the current version of the wallet. The https://docs.elrond.com/wallet/web-wallet/ still has some very old menuItems.

Reply:
added